examples/addition.json

```diff
@@ -2,25 +2,29 @@
   "nodes": [
     {
       "type": "ConstantInteger",
+      "version": "0.4.0",
       "id": "a",
       "params": {
         "value": 42
       }
     },
     {
       "type": "ConstantInteger",
+      "version": "0.4.0",
```

Comment:
Would've liked to see testing that checks and allows / denies incompatible versions?